<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="line_latedit">
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>21</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Latitude (Широта)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_lonedit">
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>21</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Longitude (Долгата)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_stop_Button">
         <property name="text">
          <string>Определить</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Задержка (сек)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditDelay">
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
